### ISSUE TITLE
Install plan / executable name (Feature request)

### ISSUE THREAD

**Issue by jgarte** [mizlan/guitar-scale-chart#3]
Instead of calling the script like this:

```
perl guitar.pl Db major
```

It would be nice to call it like this:

```
guitar-scale-chart Db major
```

We should probably have an install plan in the Makefile to copy and rename the script to an install directory.

**Comment by mizlan** [comment]
Good idea! I have never written a Makefile in my life :) but am happy to
learn

**Comment by jgarte** [comment]
I'll have to learn myself also ;)

I think the Makefile that perl expects has a different format from the standard Makefile that most people know of.

**Comment by jgarte** [comment]
Found this: https://github.com/k4rthik/git-cal/blob/master/Makefile.PL

Looks simple enough

wdyt

**Comment by mizlan** [comment]
Oh, cool! I'll toy with it for a bit; if you'd like to make a PR I'd be
very appreciative of that!

**Comment by fwcd** [comment]
I would also recommend making the script executable and adding a shebang line to make it easy to people pulling the repo to run it:

```diff
diff --git a/guitar.pl b/guitar.pl
old mode 100644
new mode 100755
index ebdb1e2..9ab0b51
--- a/guitar.pl
+++ b/guitar.pl
@@ -1,3 +1,5 @@
+#!/usr/bin/env perl
+
 use List::Util qw(any);
 use List::MoreUtils qw(first_index);
 use Term::ANSIColor;
```

That way, anyone who wishes to package it can just symlink it to `/usr/bin` or somewhere else.

**Comment by mizlan** [comment]
@jgarte I noticed you have a fork with some niceties (including what @fwcd mentions here). Would you like to upstream those changes?

**Comment by jgarte** [comment]
Sure, give me some time and I'll do that.
